ota-plus-web/app/reactapp-new/webpack.config.js

```diff
@@ -31,24 +31,11 @@ module.exports = {
         test: /\.scss$/,
         use: [
           {
-            loader: 'file-loader',
-            options: {
```

Comment:
Why some many things have been removed?

Reply:
file-loader into style-loader -> the part is configuration of file loader.